.editorconfig

```diff
@@ -0,0 +1,15 @@
+# This file is for unifying the coding style for different editors and IDEs
+# editorconfig.org
+
+root = true
+
+[*]
+indent_style = tab
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.{json,yml}]
```

Comment:
You're right, that's in the spec: http://yaml.org/spec/current.html#id2519916 - apparently the amount doesn't matter, but sticking with `2` makes sense.